src/coreclr/jit/codegencommon.cpp

```diff
@@ -1733,55 +1733,31 @@ void CodeGen::genGenerateMachineCode()
         printf(" for ");
 
 #if defined(TARGET_X86)
-        if (compiler->info.genCPU == CPU_X86)
+        printf("generic X86 CPU");
```

Comment:
`X86` requires SSE2 and the also supports the VEX/EVEX encoding so it would be nice for this path to also print `X86 CPU with AVX512` and `X86 CPU with AVX`